main.go

```diff
@@ -39,7 +38,7 @@ var (
 	configPath = "config.yml"
 
 	defaultConfig = Config{
-		PublicDir:           "./public", // left for legacy purposes
+		PackagesPaths:       []string{"./packages"},
```

Comment:
@mtojek One reason I stumbled over making this the default is because of the reference config file :-)